src/services/leafletLayerHelpers.js

```diff
@@ -55,6 +55,14 @@ angular.module('ui-leaflet')
                 return L.tileLayer(url, params.options);
             }
         },
```

Comment:
All good.  I didn't know jack about ```decorator```, so at least I learned some stuff in the process.  Thanks @nmccready @elesdoar